wire-ios-data-model/Source/Public/ZMMessage.h

```diff
@@ -162,6 +164,7 @@ typedef NS_ENUM(int16_t, ZMParticipantsRemovedReason) {
 @property (nonatomic, nullable) id <ZMSystemMessageData> parentMessage;
 @property (nonatomic, readonly) BOOL userIsTheSender;
 @property (nonatomic, nullable) NSNumber *messageTimer;
+@property (nonatomic, nullable) NSArray<NSString *> *domains;
```

Comment:
Why do we need these domains? To show them in the UI?

Reply:
Yes, we need domains to display them on UI:
`The backends **foma.wire.link** and **bella.wire.link** stopped federating.`
I believe we need to store them, but the name of the property probably is not very meaningful.